base/applications/notepad/lang/ro-RO.rc

```diff
@@ -70,7 +74,7 @@ BEGIN
     BEGIN
         MENUITEM "&Manual…", CMD_HELP_CONTENTS
         MENUITEM SEPARATOR
-        MENUITEM "&About Notepad", CMD_HELP_ABOUT_NOTEPAD
+        MENUITEM "&Despre carnețel", CMD_HELP_ABOUT_NOTEPAD
```

Comment:
`Despre Notepad` is more fitting IMHO.

Reply:
I looked down the document and Notepad was translated with Carnețel. This is why I wrote carnețel, but I can change it if you want.

Reply:
Please take care of consistency across the body of translated ReactOS resources, do not leave it as half-measure decision. So, either follow George's suggestion all the way by taking care of all the other places where "Carnețel" needs to be changed into "Notepad" (like [this one](https://github.com/reactos/reactos/blob/master/media/inf/shortcuts.inf#L1315), there maybe more), or decide on just keeping it translated as "Carnețel".